test-tshirt.cpp

```diff
@@ -0,0 +1,12 @@
+#include <assert.h>
+#include <iostream>
+#include "./tshirts.h"
+#include "./test-tshirt.h"
+void testTshirt() {
+  assert(size(37) == 'S');
+  assert(size(40) == 'M');
+  assert(size(43) == 'L');
+  assert(size(38) == 'M');

```

Comment:
What's the reason to choose 38 to be `M` and not `S`?
What's the reason from a user's perspective?